### ISSUE TITLE
Immutable size / length ??

### ISSUE THREAD

**Issue by bleucitron** [anthill/6bin#14]
in `server/src/main.js`, i try to log the size of an Immutable `List<BinData>`.

`console.log('SAVING BIIIIIIIIINS !!!', data.action.bins.length, data.action.bins.size);`

I get a TS warning: `server/src/main.ts(46,60): error TS2339: Property 'length' does not exist on type 'List<BinData>'.`
The `length` outputs the correct length, while `size` outputs `undefined`...

**Comment by DavidBruant** [comment]
blargh. Probably wrong tsd :-/

**Comment by bleucitron** [comment]
No, what's weird is i should be able to output the `size`, since it's an Immutable... I don't really care about the TS warning.
It's as if my Immutable List acted as a regular Array...

**Comment by DavidBruant** [comment]
hmm... Try `Array.isArray(data.action.bins)` ?
